<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open Rodent's Revenge - Official Launcher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon</normaloff>:/icon</iconset>
  </property>
  <widget class="QWidget" name="mainWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>60</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Level file :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="browseEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="browseButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QPushButton" name="editorButton">
         <property name="toolTip">
          <string>Launch the level editor</string>
         </property>
         <property name="text">
          <string>Editor</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="gameButton">
         <property name="toolTip">
          <string>Launch the game</string>
         </property>
         <property name="text">
          <string>Game</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Game Settings</string>
         </property>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>121</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Set number of...</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>171</width>
            <height>81</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QCheckBox" name="defineCatsNumberBox">
               <property name="text">
                <string>cats :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="numberOfCatsSpinBox">
               <property name="suffix">
                <string notr="true"/>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="maximum">
                <number>999</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QCheckBox" name="defineRWNumberBox">
               <property name="text">
                <string>random walls :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="numberOfRWSpinBox">
               <property name="suffix">
                <string notr="true"/>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="maximum">
                <number>999</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QCheckBox" name="defineLivesNumberBox">
               <property name="text">
                <string>lives (player)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="numberOfLivesSpinBox">
               <property name="suffix">
                <string notr="true"/>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="maximum">
                <number>999</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QCheckBox" name="towerModeBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>171</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Launch in 'Tower' mode</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Editor Settings</string>
         </property>
         <widget class="QCheckBox" name="emptyLevelBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>111</width>
            <height>21</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>Start with an empty level</string>
          </property>
          <property name="text">
           <string>Empty level</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="levelXSpinBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>42</width>
            <height>22</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>Horizontal size (width)</string>
          </property>
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
         <widget class="QSpinBox" name="levelYSpinBox">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>90</y>
            <width>42</width>
            <height>22</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>Vertical size (height)</string>
          </property>
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>46</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>X size</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>70</y>
            <width>46</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Y size</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="noWarningAtSaveBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>261</width>
            <height>51</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>WARNING : can be dangerous, caution when it is enabled</string>
          </property>
          <property name="text">
           <string>No warning at save</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/warning</normaloff>:/warning</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>13</height>
           </size>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>120</y>
            <width>151</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>+ See game settings</string>
          </property>
         </widget>
         <widget class="QPushButton" name="resetButton">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>90</y>
            <width>71</width>
            <height>23</height>
           </rect>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
         <zorder>levelXSpinBox</zorder>
         <zorder>levelYSpinBox</zorder>
         <zorder>label_4</zorder>
         <zorder>label_5</zorder>
         <zorder>noWarningAtSaveBox</zorder>
         <zorder>label_6</zorder>
         <zorder>resetButton</zorder>
         <zorder>emptyLevelBox</zorder>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>150</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Global Settings</string>
       </property>
       <widget class="QLineEdit" name="browseLesEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>378</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="browseLesButton">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>40</y>
          <width>25</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>181</width>
          <height>93</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="debugModeBox">
           <property name="toolTip">
            <string>FPS number will be print (F12 to toggle in game)</string>
           </property>
           <property name="text">
            <string>Debug mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="enableVSyncBox">
           <property name="toolTip">
            <string>Wait for vertical synchronisation</string>
           </property>
           <property name="text">
            <string>Enable V.Sync.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="adjustWindowSizeBox">
           <property name="toolTip">
            <string>The window will expand to adjust to the level size (limited by screen's resolution)</string>
           </property>
           <property name="text">
            <string>Adjust window size</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QCheckBox" name="defineFpsLimitBox">
             <property name="toolTip">
              <string>This will limit the FPS (0 = no limit)</string>
             </property>
             <property name="text">
              <string>Set FPS  limit :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="fpsLimitSpinBox">
             <property name="suffix">
              <string notr="true"/>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>99999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="editModsButton">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>70</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Edit mods to use</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="useLesBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Use this LES description file</string>
        </property>
        <property name="text">
         <string>Use LES file</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="myPcSucksBox">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>110</y>
          <width>231</width>
          <height>16</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>If the game seems to have a graphical problem, try to check this case (performances may be reduced)</string>
        </property>
        <property name="text">
         <string>Enable compatibility mode</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Launcher</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionFrench"/>
    </widget>
    <widget class="QMenu" name="menuGame">
     <property name="title">
      <string>Game</string>
     </property>
     <addaction name="actionLocation"/>
    </widget>
    <addaction name="menuGame"/>
    <addaction name="menuLanguage"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionFrench">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Français</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionLocation">
   <property name="text">
    <string>Location</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
